google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -628,6 +629,8 @@ def _download_table_bqstorage(
     # is available and can be imported.
     from google.cloud import bigquery_storage
 
+    _verify_bq_storage_version()
```

Comment:
Why is this fatal here, but a warning elsewhere?

Reply:
Similar reasoning as below, the helper's job is to download data specifically using the BQ Storage API. If it is called with an obsolete `bqstorage_client`, the actual error most likely is somewhere higher up.

Now, the check _might_ be unnecessary if the higher level code is water proof, but that might not be the case.